Sources/XS2AiOS/XS2AiOS.swift

```diff
@@ -94,7 +87,7 @@ extension XS2AiOS {
 			wizardSessionKey: String,
 			backButtonAction: @escaping () -> Void = {},
 			onStepChanged: @escaping (WizardStep?) -> Void = {_ in },
-			onRender: @escaping (WizardStep?) -> Void = {_ in },
+			onRender: @escaping () -> Void = { },
```

Comment:
Hi, thanks for making those changes so quickly :)

Let me give you some background why we need those changes. 
Basically, we need `onRender`, because the `currentStep` is not giving us enough information about the current state of the form. The problem i'm facing is related to the state where the `currentState` is `login` and the `step` (or `currentStepNormalized`) is `login_next`, or any other scenario where those 2 are out of sync. 
I'm not able to implement back button behaviour properly when the last state received from `onStepChanged` is `login` and the form is displaying `step` = `login_next`. When user press back button, i should trigger `goBack()`, but i was not informed that user is already on second page of the form.

To be able to use the `onRender` method the `step` property is needed. It would be awesome to have it back, like in the initial commit.

Reply:
Correct me if I'm wrong:

If you are on `login_next`, you want to `goBack()`, but on `login` you want to hide everything from this SDK and show your own search / screen?

Reply:
Exposing the `currentStep` gives us some flexibility in implementation. I'm not sure if checking `isLogin` is enough, as we need similar handling on bank search screen. If, by any chance, the bank search is displayed, we need to handle back button the same way as on login screen(close the screen). Check the example 2 from my previous post.
We can handle everything we need with the `onStepChanged`, `isLogin` and `isBankSearch`.

Our current code simply checks:
```
if XS2AiOS.shared.currentStep == .login {
    close()
} else {
    xs2aViewController?.goBack()
}
```
We can change it to `if isLogin || isBankSearch`.

Reply:
@ziper1 I can also implement in addition a `isBank` method, that follow the same logic for the bank search screen. This is also what the other customer uses.

The reason why I prefer that to the steps is that there are about ~12 steps in total (the 4 current ones are just an abstraction) and I would like to not have a customer implementation based on those, as there are not really part of any public API. So I would like to keep them `private` and remain in control what step name is behind `isLogin` or `isBank` and so on.

Reply:
Thank you. For me it's enough.
Having `isBank` and `isLogin` allows me to implement features i need.

Reply:
@ziper1 Awesome, I updated the PR. Let me know if this works, maybe you can check this out locally and test it. Then I can merge it :)